### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by Krishnan-Priyanshu**
Hi Team,

Please review this PR, we have a coverage percentage of 83.1%
image

### CODE REVIEW

File: powerflex/sds_datasource_test.go [hunk 1]
```diff
@@ -68,35 +68,35 @@ func TestSdsDataSource(t *testing.T) {
 
 var SdsDataSourceConfig1 = `
 data "powerflex_sds" "example1" {
-	protection_domain_id = "4eeb304600000000"
-	sds_names = ["node1"]
+	protection_domain_id = "202a046600000000"
+	sds_names = ["SDS_10.247.100.214"]
```
Comment (AnikaAgiwal2711) [inline powerflex/sds_datasource_test.go:72]: This name doesn't seem good to me .

File: powerflex/storagepool_data_source_test.go [hunk 2]
```diff
@@ -17,16 +17,16 @@ func TestStoragePoolDataSource(t *testing.T) {
 					resource.TestCheckResourceAttr("data.powerflex_storage_pool.example1", "storage_pools.#", "2"),
 					resource.TestCheckResourceAttr("data.powerflex_storage_pool.example1", "storage_pools.0.name", "pool2"),
 					resource.TestCheckResourceAttr("data.powerflex_storage_pool.example1", "storage_pools.1.name", "pool1"),
-					resource.TestCheckResourceAttr("data.powerflex_storage_pool.example1", "protection_domain_id", "4eeb304600000000"),
+					resource.TestCheckResourceAttr("data.powerflex_storage_pool.example1", "protection_domain_id", "202a046600000000"),
```
Comment (AnikaAgiwal2711) [inline powerflex/storagepool_data_source_test.go:20]: Also I believe that for ids or names , it's better to have that as part of variables(environment variable will also work I believe).

File: powerflex/sdc_datasource_test.go [hunk 3]
```diff
@@ -23,7 +23,7 @@ func init() {
 	sdcTestData.noOfSdc = "1"
 	sdcTestData.noOflinks = "4"
 	sdcTestData.name = ""
-	sdcTestData.sdcip = SdsResourceTestData.SdcIP
+	sdcTestData.sdcip = "10.247.66.194"
```
Comment (shenda1) [inline powerflex/sdc_datasource_test.go:26]: This can be read from the environment variable

File: powerflex/sdc_resource_test.go [hunk 4]
```diff
@@ -22,11 +22,11 @@ var sdcResourceTestData resourceDataPoints
...
 	sdcResourceTestData.noOflinks = "4"
-	sdcResourceTestData.name = "powerflex_sdc26"
-	sdcResourceTestData.newname = ""
-	sdcResourceTestData.sdcguid = "0877AE5E-BDBF-4E87-A002-218D9F883896"
-	sdcResourceTestData.sdcip = ""
-	sdcResourceTestData.systemid = "0e7a082862fedf0f"
+	sdcResourceTestData.name = "terraform_sdc_create"
+	sdcResourceTestData.newname = "terraform_rename"
+	sdcResourceTestData.sdcguid = "C87ACC43-298B-4AD3-A95F-344FE83192C6"
+	sdcResourceTestData.sdcip = "10.247.66.67"
```
Comment (shenda1) [inline powerflex/sdc_resource_test.go:28]: This can be read from the environment variable

**Comment by Krishnan-Priyanshu** [comment]
Deleting the branch and will be creating new PR for dev-1.1.0 branch

**Pull request closed without merging**